cli/azd/pkg/apphost/aca_ingress.go

```diff
@@ -0,0 +1,172 @@
+package apphost
+
+import (
+	"fmt"
+)
+
+// buildAcaIngress builds the Azure Container Apps ingress configuration from the provided bindings.
```

Comment:
I feel like some high level description of the algorithm here might be useful - We end up doing multiple levels of grouping here, it might be helpful to orient the reader about what's about to happen as we try to merge multiple bindings together to build the ACA ingress configuration.